zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -99,8 +99,13 @@ public static void main(String[] args) throws Exception {
     jettyServer.setHandler(contexts);
 
     LOG.info("Start zeppelin server");
-    jettyServer.start();
-    LOG.info("Started");
+    try {
+      jettyServer.start();
+    } catch (Exception e) {
+      LOG.error("Error while running jettyServer", e);
```

Comment:
Thanks for pointing out.
I think in this case it is correct and appropriate - it is however fairly easy to mix up and miss the exception stack: http://www.slf4j.org/faq.html#paramException
But in any case we should use the logger in any place possible ;)